client/src/components/Ticketing/checkout/CompleteOrderForm.tsx

```diff
@@ -184,35 +184,43 @@ export default function CompleteOrderForm(
             border-zinc-300 p-4 rounded-lg " type='text' name="visit-source" onChange={(ev: React.ChangeEvent<HTMLInputElement>): void =>
                       setvisitSource(ev.target.value)} placeholder="How did you hear about us?"/>
                   </div>
-
-
-                  <input className="input w-full  border
-            border-zinc-300 p-4 rounded-lg col-span-2 " type='text' name="comments" onChange={(ev: React.ChangeEvent<HTMLInputElement>): void =>
-                    setComments(ev.target.value)} placeholder="Comments"/>
+                  <div>
+                    <span className=" block text-sm font-medium text-slate-700 ml-1">
+                    Seating Accommodations
+                    </span>
+                    <select className="input w-full  border
+            border-zinc-300 p-4 mt-1 rounded-lg col-span-2 " name="seatingAcc" value={seatingAcc} onChange={(ev: React.ChangeEvent<HTMLSelectElement>): void =>
+                      setseatingAcc(ev.target.value)}>
+                      <option value="None">No, not at this time</option>
+                      <option value="Wheel Chair">Yes, wheelchair seat(s)</option>
+                      <option value="Aisle Seat">Yes, aisle seat(s)</option>
+                      <option value="First/Ground floor">Yes, seat(s) on the ground or the first level</option>
+                      <option value="ASL Interpreter">Yes, seats in the ASL interpreters section</option>
+                      <option value="Wide Seats">Yes, wide seats</option>
+                      <option value="Other">Yes, other (describe accommodation in comment box below, if not listed)</option>
```

Comment:
I just realized that the `comments` attribute of this form isn't actually being used anywhere once its sent to the API. So to accomodate for this, and to make sure that the seattingaccom is updated to whatever the comments say if the user specifies "Other". You could probably put a check in the `doCheckout()` function that if the seattingaccom is "Other" and comments is not undefined, then to set seattingaccom to the comments. Let me know if you have any questions.

Reply:
fixed.  added functions to handle seatingacc and comments to react element in completeorderform.  Will assign comment value to seatingacc only if "other" is selected for seatingacc.
![image](https://github.com/WonderTix/WonderTix/assets/97830813/4e21940c-66ff-4d5f-901b-c2d5afa5385e)

Reply:
updated handleSeatingAccChange, removed the if statement.  Want to keep any comments made by customer, even if they dont chose "Other"
![image](https://github.com/WonderTix/WonderTix/assets/97830813/8307c182-5860-4074-b9b8-ab06cb5dffb2)

Reply:
![image](https://github.com/WonderTix/WonderTix/assets/97830813/0809436f-276e-49bd-b1d8-499159f65152)